### DISCUSSION TITLE
Dotted stroke dot length

### DISCUSSION THREAD

**Discussion by daretofly1** [photopea/photopea#7338] [Ideas]
When we choose the dash option for a stroke, we are able to change the spacing or dot length between the dashes. But the Dotted stroke has no such feature, is there a way to add the spacing option like in the brush options?

**Comment by photopea** [comment]
There is a text field "dashes", where you can enter the dash pattern. It is a list of numbers, it is described here:

https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/setLineDash#:~:text=An%20Array%20of%20numbers%20that%20specify%20distances

or here

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dasharray#:~:text=A%20list%20of%20comma%20and/or%20white%20space

There are even examples at the bottom of the panel, so use them to see how it works.
